Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -1373,6 +1373,94 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
                        _("Objects"),
                        "res/actions/layer.png")
       .AddParameter("object", _("Object"));
+  
+  obj.AddAction("RemoveEffect",
+                _("Remove an effect"),
+                _("Remove an effect from the object"),
+                _("Remove _PARAM1_ effect from the _PARAM0_"),
+                _("Effects"),
+                "res/actions/effect24.png",
+                "res/actions/effect.png")
+      .AddParameter("object", _("Object"))
+      .AddParameter("string", _("Effect Name"))
+      .MarkAsSimple();
+  
+  obj.AddAction("EnableEffect",
+                _("Enable an object effect"),
+                _("Enable an effect on the object"),
+                _("Enable _PARAM1_ effect on _PARAM0_ : _PARAM2_"),
+                _("Effects"),
+                "res/actions/effect24.png",
+                "res/actions/effect.png")
+      .AddParameter("object", _("Object"))
+      .AddParameter("string", _("Effect Name"))
+      .AddParameter("yesorno", _("Enable"))
+      .MarkAsSimple();
+  
+  obj.AddAction("SetEffectDoubleParameter",
+                _("Set a double parameter for effect"),
+                _("Set a double parameter for an effect of the object"),
+                _("Set the value of _PARAM2_ as _PARAM3_ for the " 
+                  "effect _PARAM1_ of object _PARAM0_"),
+                _("Effects"),
+                "res/actions/effect24.png",
+                "res/actions/effect.png")
+      .AddParameter("object", _("Object"))
+      .AddParameter("string", _("Effect Name"))
+      .AddParameter("string", _("Parameter Name"))
+      .AddParameter("number", _("Parameter Value"))
```

Comment:
Follow the exact same namings as for layers:
```suggestion
      .AddParameter("string", _("Effect"))
      .AddParameter("string", _("Parameter name"))
      .AddParameter("expression", _("New value"))
```